src/contracts/core/AllocationManager.sol

```diff
@@ -403,6 +438,27 @@ contract AllocationManager is
         return (freeMagnitudeToAdd, nextIndex);
     }
 
+    /**
+     * @notice Returns the allocation delay of an operator
+     * @param operator The operator to get the allocation delay for
+     * @dev Defaults to `DEFAULT_ALLOCATION_DELAY` if none is set
+     */
+    function allocationDelay(
+        address operator
+    ) public view returns (uint32 delay) {
+        AllocationDelayInfo memory delayInfo = _allocationDelayInfo[operator];
+
+        bool pendingInEffect = delayInfo.pendingDelay != 0 && block.timestamp >= delayInfo.pendingDelayEffectTimestamp;
+
+        if (delayInfo.delay == 0 && !pendingInEffect) {
+            return DEFAULT_ALLOCATION_DELAY;
```

Comment:
Per comment above, lets get rid of `DEFAULT_ALLOCATION_DELAY` and this if block here. We also want to allow configurging 0 delay anyways without enforcing a default.